### PR TITLE
Use local jsontest service

### PR DESCRIPTION
**Pull request by webrouse**
Jira: https://keboola.atlassian.net/browse/COM-596

### CODE REVIEW

File: tests/phpunit/data/requestCache/config.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
       "ttl": 7200
     },
     "api": {
-      "baseUrl": "http://time.jsontest.com/",
+      "baseUrl": "http://jsontest",
```
Comment (webrouse) [inline tests/phpunit/data/requestCache/config.json:7]: Sluzba http://time.jsontest.com ma vypadok.

image

Pouziva sa v 3 testoch, vracia to datum ako JSON response.
Premigroval som to na lokalnu Docker sluzbu, ktora robi to iste.

File: docker/jsontest/JsonTests.pl [hunk 2]
```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env perl
```
Comment (webrouse) [inline docker/jsontest/JsonTests.pl:1]: Tento script som nasiel na nete, a robi to iste ako nefunkcna sluzba http://time.jsontest.com/

**Review by webrouse — COMMENTED** [review]
Vid komenty

**Review by ondrajodas — APPROVED** [review]
ok

**Pull request merged**